### DISCUSSION TITLE
Please add wake word.

### DISCUSSION THREAD

**Discussion by BrandgrandRealMe** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#527] [Ideas]
Please add "Ok, Google" and "Hey Google" support!
This should be important. 

Everyone:

> If you say yes, State the reasoning behind why you want It.
> If you say no, State the reasoning behind why you don't think it should be added.

Here is mine: Yes, Cause I need it and Cortana reacts to ALMOST EVERY SOUND.

**Comment by im-coder-lg** [comment]
It is already added. You will have to build the app yourself.

**Comment by im-coder-lg** [comment]
Hey @BrandgrandRealMe this is now released in 1.0.0, you can compile the app yourself/get the compiled installer.
